.github/workflows/ci.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: package tasks ipa
         working-directory: ./apps/tasks/
         run: |
-          flutter build ios --no-codesign
+          flutter build ios --no-codesign #--debug
```

Comment:
`#--debug`?

Reply:
We need that later on... If we build an ios app for release, we need to specify the debug flag (see android)